README.md

```diff
@@ -296,7 +296,7 @@ Symptoms:
 Logging into Openshift with the SAML provider will have a 403 Forbidden Error.
 On the saml-auth pod (only the debug version) running on Openshift, you will see this line in `/var/log/httpd/mellon_diagnostics`
 `am_check_permissions processing condition 0 of 1: varname="user" flags=[REG] str=".+" directive="MellonCond user .+ [REG]" failed (no OR condition) returning HTTP_FORBIDDEN`
-This error is an indication that your SAML mappings in Keycloak are not mapping correctly. Ensure that the property `username` from keycloak is mapped to the SAML Attribute Name `user`. This involves setting a mapper in the client in Keycloak. See [testing_with_keycloak.md](keycloak/testing_with_keycloak.md#mapping-the-data-from-keycloak-to-mod_auth_mellon)
+This error is an indication that your SAML mappings coming from your IdP are not mapping correctly. The SAML property needs to be mapped as "user", whereas some identity providers, like keycloak, send this property as "username". See [testing_with_keycloak.md](keycloak/testing_with_keycloak.md#mapping-the-data-from-keycloak-to-mod_auth_mellon)
```

Comment:
@bontreger keycloack actually doens't send any properties by default it just expects you will look at the NAME_ID field and use that by default which this code currently doesn't, it expects the `user` property. I would just drop everything from the "whereas some" on.